incubator/command-generate/src/main/moditect/module-info.java

```diff
@@ -29,13 +29,15 @@
 
     requires com.fasterxml.jackson.dataformat.yaml;
     requires com.fasterxml.jackson.databind;
+    requires org.apache.commons.collections4;
```

Comment:
We typically try to use either built-in JDK collections or Agrona collections instead of commons collections, what part of commons collections are we using in this implementation?

(trying to minimize dependency sprawl)

Reply:
I use the helper methods `MapUtils.isNotEmpty` and `CollectionUtils.isNotEmpty`, they are shortcuts for checking if the argument is not null and is not empty. It came handy in some places and apache commons was already added as a dependency so I assumed it's fine to use. But I can expand the null+emptiness checks, and remove the use of apache commons, if you prefer, not an issue. Pls LMK.